themes/default/content/docs/pulumi-cloud/esc/providers/_index.md

```diff
@@ -12,7 +12,7 @@ menu:
 
 Pulumi ESC Providers enable you to dynamically import Secrets and Configuration from the provider into your Environment.
```

Comment:
Since these aren't proper names, they should be lowercase:

```suggestion
Pulumi ESC providers enable you to dynamically import secrets and configuration into your environment.
```